Marlin/pins_PRINTRBOARD_REVF.h

```diff
@@ -68,6 +71,28 @@
 
 #define BOARD_NAME         "Printrboard Rev F"
 #define LARGE_FLASH        true
+#ifndef USBCON
+  #error "USBCON should be defined by the platform for this board."
+#endif
+
```

Comment:
I just tested each of the other boards: TEENSYLU, TEENSY2, SAV_MKI, PRINTRBOARD, BRAINWAVE_PRO, 5DPRINT, (and of course PRINTRBOARD_REVF) with USBCON define removed from respective pins_[board].h and all compiled w/o issue.

It seems removal of USBCON define from all pins_[board].h would be appropriate, allowing serial.h to pull it in as appropriate from Arduino.h ?